apps/aecore/lib/aecore/chain/block.ex

```diff
@@ -24,11 +24,39 @@ defmodule Aecore.Chain.Block do
     @version
   end
 
+  # @spec genesis_header() :: Header.t()
+  # defp genesis_header do
+  #   header = Application.get_env(:aecore, :pow)[:genesis_header]
+  #   struct(Header, header)
+  # end
+
+  def genesis_header do
```

Comment:
Why create this function if there is one already doing this in the [Genesis module](https://github.com/aeternity/elixir-node/blob/master/apps/aecore/lib/aecore/chain/genesis.ex#L71)?

Reply:
As I said there is some stuff that needs to be removed, I'll take care of it

Reply:
I didn't know you are still working on this and aware of the issue in hand. I guess there was some miscommunication on my part.